README.md

```diff
@@ -26,7 +26,13 @@ creating monitors for them and sending pings when tasks run, succeed, or fail. A
 
 Requires Celery 4.0 or higher. Celery auto-discover utilizes the Celery [message protocol version 2](https://docs.celeryproject.org/en/stable/internals/protocol.html#version-2).
 
-> Note: tasks on [solar schedules](https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#solar-schedules) are not supported and will be ignored.
+<details>
```

Comment:
this is blowing my mind...what is this tag?!?!?!?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details